src/main/java/testUI/elements/TestUI.java

```diff
@@ -97,11 +97,11 @@ protected String  getStringElement(String accesibilityIdiOS, String accesibility
     protected String getLocator(String accesibilityIdiOS, String accesibilityId) {
         if (Configuration.automationType.equals(Configuration.IOS_PLATFORM)) {
             if (accesibilityIdiOS != null && !accesibilityIdiOS.isEmpty())
-                return accesibilityIdiOS.split(": ")[1];
+                return accesibilityIdiOS.split(": ", 2)[1];
```

Comment:
The user could write a locator like `"Incorrect password! Expected: to match: ^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[@$!%*#?&])[A-Za-z0-9@$!%*#?&]{8,}` and it will split it like this  
```
String[2] { "Incorrect password! Expected", "to match: ^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[@$!%*#?&])[A-Za-z0-9@$!%*#?&]{8,}$" }
```

Is that the expected output?

Reply:
the way testUI works is that it has a prefix with the form of `locatorStrategy: elementLocator` so, what we need to do is to split the string into two: locatorStrategy and elementLocator. You can look at that in the next lines:

```
switch (locator.split(": ", 2)[0]) {
            case "accessibilityId":
```